src/modules/10-common/components/MultiTypeDelegateSelector/MultiTypeDelegateSelector.tsx

```diff
@@ -31,6 +31,7 @@ export interface MultiTypeDelegateSelectorProps extends IFormGroupProps {
   allowableTypes?: MultiTypeInputType[]
   tooltipProps?: DataTooltipInterface
   inputProps: Omit<DelegateSelectorsProps, 'onChange'>
+  formik?: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9248504)
```suggestion
  formik?: unknown
```